assistants/operations.tsp

```diff
@@ -0,0 +1,161 @@
+import "@typespec/http";
+import "@typespec/openapi";
+
+import "../common/errors.tsp";
+import "./models.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace OpenAI;
+
+@route("/assistants")
+interface Assistants {
+  @post
+  @operationId("createAssistant")
+  @tag("Assistants")
+  @summary("Create an assistant with a model and instructions.")
+  createAssistant(
+    @body assistant: CreateAssistantRequest,
+  ): AssistantObject | ErrorResponse;
+
+  @get
+  @operationId("listAssistants")
+  @tag("Assistants")
+  @summary("Returns a list of assistants.")
+  listFiles(
+    /** 
+     * A limit on the number of objects to be returned. Limit can range between 1 and 100, and the
+     * default is 20.
+     */
+    @query limit?: int32 = 20;
+
+    /**
+     * Sort order by the `created_at` timestamp of the objects. `asc` for ascending order and`desc`
+     * for descending order.
+     */
+    @query order?: "asc" | "desc" = "desc";
```

Comment:
@bterlson : I noticed that the generated OpenAPIv3 spec seems to have an issue when generating some enums. For example, this query parameter produces the following:

```text
- name: order
  in: query
  required: false
  description: |-
    Sort order by the `created_at` timestamp of the objects. `asc` for ascending order and`desc`
    for descending order.
  schema:
    type: string
    enum:
      - asc
      - desc
      - desc
      - desc
      - desc
      - desc
      - desc
    default: desc
```

Please let me know where I can file a bug for this one. 🙂